tests/projects/sbt-service/deployment/project.yml

```diff
@@ -1,11 +1,13 @@
 name: 'sbtservice'
```

Comment:
We should find a way to centralise our yaml formatting

Reply:
Maybe we could use prettier https://prettier.io/blog/2018/07/29/1.14.0.html

Reply:
Hmmm, prettier can't be configured. The only way I've found in which `ruaml` alters the formatting in roundtrip mode is by removing spaces around arrays `[ 'MPyl' ]` becomes `['MPyl']`